src/go/rpk/pkg/cli/cmd/root.go

```diff
@@ -425,3 +430,45 @@ func walk(c *cobra.Command, f func(*cobra.Command)) {
 		walk(c, f)
 	}
 }
+
+func wrappedLocalFlagUsages(cmd *cobra.Command) string {
+	width := 80
+	if ws, err := mTerm.GetWinsize(0); err == nil {
+		width = int(ws.Width)
+	}
+	return cmd.LocalFlags().FlagUsagesWrapped(width - 1)
+}
+
+func wrappedGlobalFlagUsages(cmd *cobra.Command) string {
+	width := 80
+	if ws, err := mTerm.GetWinsize(0); err == nil {
+		width = int(ws.Width)
+	}
+	return cmd.InheritedFlags().FlagUsagesWrapped(width - 1)
+}
+
+// This is the same Cobra usage template but using the wrapped flag usages.
+var usageTemplate = `Usage:{{if .Runnable}}
+  {{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
+  {{.CommandPath}} [command]{{end}}{{if gt (len .Aliases) 0}}
+
+Aliases:
+  {{.NameAndAliases}}{{end}}{{if .HasExample}}
+
+Examples:
+{{.Example}}{{end}}{{if .HasAvailableSubCommands}}
+
+Available Commands:{{range .Commands}}{{if (or .IsAvailableCommand (eq .Name "help"))}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableLocalFlags}}
+
+Flags:
+{{ wrappedLocalFlagUsages . | trimRightSpace}}{{end}}{{if .HasAvailableInheritedFlags}}
+
+Global Flags:
+{{ wrappedGlobalFlagUsages . | trimRightSpace}}{{end}}{{if .HasHelpSubCommands}}
+
+Additional help topics:{{range .Commands}}{{if .IsAdditionalHelpTopicCommand}}
+  {{rpad .CommandPath .CommandPathPadding}} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableSubCommands}}
+
+Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
+`
```

Comment:
Same as Cobra's: https://github.com/spf13/cobra/blob/main/command.go#L539-L568

With the change of the wrappedFlagUsages.